src/decorators/getValidators.integration.test.ts

```diff
@@ -0,0 +1,75 @@
+import { describe, it, expect } from 'vitest';
+import { Address, createPublicClient, http } from 'viem';
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts';
+
+import { nitroTestnodeL2 } from '../chains';
+import { rollupAdminLogicPublicActions } from './rollupAdminLogicPublicActions';
+import { getInformationFromTestnode, getNitroTestnodePrivateKeyAccounts } from '../testHelpers';
+import { getValidators } from '../getValidators';
+
+const { l3RollupOwner } = getNitroTestnodePrivateKeyAccounts();
+const { l3Rollup, l3UpgradeExecutor } = getInformationFromTestnode();
+
+const client = createPublicClient({
+  chain: nitroTestnodeL2,
+  transport: http(),
+}).extend(
+  rollupAdminLogicPublicActions({
+    rollup: l3Rollup,
+  }),
+);
+
+async function toggleValidator(validator: Address, state: boolean) {
+  const tx = await client.rollupAdminLogicPrepareTransactionRequest({
+    functionName: 'setValidator',
+    args: [[validator], [state]],
+    account: l3RollupOwner.address,
+    upgradeExecutor: l3UpgradeExecutor,
+    rollup: l3Rollup,
+  });
+
+  await client.sendRawTransaction({
+    serializedTransaction: await l3RollupOwner.signTransaction(tx),
+  });
+}
+
+describe('successfully get validators', async () => {
+  it('when disabling the same validator multiple time', async () => {
+    const randomAccount = privateKeyToAccount(generatePrivateKey()).address;
+
+    const initialValidators = await getValidators(client, { rollupAddress: l3Rollup });
+    // By default, nitro chains have 10 validators
```

Comment:
Where does this default come from?

Very often for Orbit chains there is only one?

Reply:
Default state when you start the nitro testnode

Reply:
can you update the comment please?